.github/workflows/unit-test.yml

```
@@ -69,9 +95,15 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: [''] # Latest WordPress version.
+                include:
+                    # Test with the previous WP version.
+                    - php: '7.4'
```

```suggestion
                    - php: '5.6'
                    - php: '7.4'
                    - php: '8.2'
```

I agree with the above, testing against lowest, highest and a midpoint would be good.

On a couple of work repos, we're moving towards [building in one workflow and then using the asset to run the tests](https://github.com/10up/ads-txt/pull/137/files), maybe that could be considered as a follow up to reduce the runtime of the actions.

---

Okay, since there's apparently strong sentiment towards including a [5.x](https://github.com/WordPress/gutenberg/pull/46983#discussion_r1113579524) and and [8.x](https://github.com/WordPress/gutenberg/pull/46983#discussion_r1113579524) version, I'll add commit this patch.

> On a couple of work repos, we're moving towards [building in one workflow and then using the asset to run the tests](https://github.com/10up/ads-txt/pull/137/files), maybe that could be considered as a follow up to reduce the runtime of the actions.

Cool, sounds good! Thank you for surfacing 😊